### ISSUE TITLE
Outputting '}}' as 'postman_variable_cl/OSE_DB'

### ISSUE THREAD

**Issue by nicoburns** [postmanlabs/swagger2-to-postman#29]
I have a swagger 2 JSON file containing the following property

` "host" : "{{base_url}}"`

Unfortunately this is getting converted to `{{base_urlpostman_variable_cl/OSE_DB` in the imported collection, presumably because a parsing token isn't getting replaced somewhere.

Affects the postman desktop app.

**Comment by ClimberBear** [comment]
Hi
Any workaround/suggestion?

**Comment by nicoburns** [comment]
@ClimberBear Yes actually, I eventually noticed that changing the variable name to some other values fixed it. So try playing around with some different var names if you can.

**Comment by ClimberBear** [comment]
Really thanks!!!!



------------------------------
*Jose M. Albarrán*
*Socio Director *
*b-Thinking Software*
Tlfo: +34 616 968 262
*e-Mail*: jmalbarran@b-Thinking.com
*LinkedIn*: https://es.linkedin.com/in/jmalbarran
Also available in WhatsApp, Telegram, Skype (jmalbarran)  and Google
Hangouts <https://hangouts.google.com/call/km5oqrxdtlo3lnawe3svlggrnea>
---------------------

2018-02-15 1:04 GMT+01:00 Nico Burns <notifications@github.com>:

**Comment by mjgoethe** [comment]
This is actually a problem in the most recent version of Postman when importing a Swagger file. It's actually due to the fact that the url.resolve function in Node would previously truncate domain parts at 63 characters and then just sort of assumed the rest belonged as part of the path (which is just kind of bonkers). This is aggravated by the fact that the Postman team chose rather long substitution strings for handling variables. The Node team fixed the issue in [this commit](https://github.com/nodejs/node/commit/c65d55f0878a42c5a9fe775ee190d6092c527661#diff-bbc5176adff1bbc01acd61bfc85d049f). Postman is using a slightly older version of Electron, so it makes sense that it still has this issue. For this repo, you should be able to fix this for yourself by running Node v8.0 or later. 

... or you can make your host name variable names 14 characters or less.

The team here might still want to reconsider their approach though, since the resolve function also converts all characters to lowercase, which is problematic for variable names.

**Comment by jehof** [comment]
This is really a dump error. I use various variables for my host-addresses, like {{gateway}}, {{internalstateservice}}, etc.

The {{gateway}}-variable is handled properly, but {{internalstateservice}} is obscured to **{{internalstateservicepostman_variable_cl/OSE_DB**. This makes it pretty useless.

I hope that they will fix it soon.
